### PR TITLE
Minor non-functional updates to sample 4

### PR DESCRIPTION
**Pull request by TaoChenOSU**
### Motivation and Context

It is not obvious to users that Sample 4 currently only processes markdown files.

### Description

1. Add a paragraph in the sample 4 readme that points out the app only processes markdown files.
2. Add a tooltip in the repo selection page that informs users only the files of the specified type will be processed.

### Contribution Checklist

- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows SK Contribution Guidelines (https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md)
- [ ] The code follows the .NET coding conventions (https://learn.microsoft.com/dotnet/csharp/fundamentals/coding-style/coding-conventions) verified with `dotnet format`
- [ ] All unit tests pass, and I have added new tests where possible
- [ ] I didn't break anyone :smile:

### CODE REVIEW

**Comment by lukins** [comment]
No no

**Review by lukins — COMMENTED** [review]
-deleted-

**Comment by lukins** [comment]
Ještě ty 3 přispěvovatelé, teď nevím co s tím, dávám pauzu

File: samples/apps/github-qna-webapp-react/README.md [hunk 1]
```diff
@@ -24,7 +24,12 @@ to get answers about it. The sample highlights how [memory](https://aka.ms/sk/me
 and [embeddings](https://aka.ms/sk/embeddings) work along with the
 [SemanticTextPartitioner](../../../dotnet/src/SemanticKernel/SemanticFunctions/Partitioning/SemanticTextPartitioner.cs)
 when the size of the data is larger than the allowed token limited.
-Each SK function will call Open AI to perform the tasks you ask about.​
+Each SK function will call Open AI to perform the tasks you ask about.
+
+This sample will create embeddings only for markdown files by default in
+order to save time and costs.  You can change the filter by going to
+[GitHubSkill.cs](/semantic-kernel/blob/memory-sample/samples/dotnet/github-skills/GitHubSkill.cs)
```
Comment (dluc) [inline samples/apps/github-qna-webapp-react/README.md:31]: incorrect link, pointing to a file in a branch. The link should be a relative path, starting with `../`

File: samples/apps/github-qna-webapp-react/README.md [hunk 2]
```diff
@@ -24,7 +24,12 @@ to get answers about it. The sample highlights how [memory](https://aka.ms/sk/me
 and [embeddings](https://aka.ms/sk/embeddings) work along with the
 [SemanticTextPartitioner](../../../dotnet/src/SemanticKernel/SemanticFunctions/Partitioning/SemanticTextPartitioner.cs)
 when the size of the data is larger than the allowed token limited.
-Each SK function will call Open AI to perform the tasks you ask about.​
+Each SK function will call Open AI to perform the tasks you ask about.
+
+This sample will create embeddings only for markdown files by default in
+order to save time and costs.  You can change the filter by going to
```
Comment (dluc) [inline samples/apps/github-qna-webapp-react/README.md:30]: "In order to reduce costs and improve the overall performance, this sample app indexes only content extracted from markdown files".

File: samples/apps/github-qna-webapp-react/README.md [hunk 3]
```diff
@@ -24,7 +24,12 @@ to get answers about it. The sample highlights how [memory](https://aka.ms/sk/me
...
+[GitHubSkill.cs](/semantic-kernel/blob/memory-sample/samples/dotnet/github-skills/GitHubSkill.cs)
+and editing the searchPattern filter.
```
Comment (dluc) [inline samples/apps/github-qna-webapp-react/README.md:32]: it's not that simple, e.g. the code can index only text files, if the pattern is `*.pdf` the solution won't work. Also the solution hasn't been tested with code, e.g. if the filter is `*.py`. I would drop this second sentence for now.

File: samples/apps/github-qna-webapp-react/src/components/GitHubRepoSelection.tsx [hunk 4]
```diff
@@ -122,9 +123,19 @@ const GitHubProjectSelection: FC<IData> = ({ uri, keyConfig, prevProject, prevBr
...
-                <strong>Embedding Filetypes</strong>
-            </Label>
+            <div style={{ display: 'flex', flexDirection: 'row' }}>
+                <Label>
+                    <strong>Embedding File Types</strong>
+                </Label>
+                <InfoLabel
+                    info={
+                        <div style={{ maxWidth: 250 }}>
+                            Only files of type below will be used for embedding and for answers.
```
Comment (dluc) [inline samples/apps/github-qna-webapp-react/src/components/GitHubRepoSelection.tsx:133]: what about "Embedding and answers will be based on files of the type indicated below" ?

**Review by dluc — CHANGES_REQUESTED** [review]

File: samples/apps/github-qna-webapp-react/src/components/GitHubRepoSelection.tsx (hunk 4 above)
Comment (evchaki) [reply samples/apps/github-qna-webapp-react/src/components/GitHubRepoSelection.tsx:133]: looks good

**Review by craigomatic — APPROVED** [review]
Looks good, LGTM

**Review by dluc — APPROVED** [review]

**Pull request merged**